src/styles/index.ts

```diff
@@ -4499,6 +4499,15 @@ const styles = (theme: ThemeColors) =>
             backgroundColor: theme.highlightBG,
         },
 
+        onboardingSmallIcon: {
+            padding: 10,
+        },
+
+        onboardingWrapper: {
+            paddingLeft: 32,
+            paddingRight: 32,
+        },
```

Comment:
Remove. Use `styles.ph8`

Reply:
I tried to use `styles.ph8` and it doesn't work.

Reply:
Ah I see. It's because `styles.sidebarLinkInner` uses `paddingLeft`/`paddingRight` and those styles has higher priority than `paddingHorizontal`. In that case let's use `[styles.pl8, styles.pr8]`.

Also there is styles.pr9 which is wrong, please rename it to styles.pr8

Reply:
Updated.